distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -231,6 +235,12 @@ void execute(Terminal terminal, String pluginId, boolean isBatch, Environment en
         install(terminal, isBatch, extractedZip, env);
     }
 
+    private static void handleInstallIngestGeoIp() throws UserException {
+        throw new UserException(
+                ExitCodes.OK,
```

Comment:
This is deliberately status code zero so as to avoid failing automation that relies on installing this plugin during deployment.